docs/DeveloperGuide.md

```diff
@@ -270,21 +270,19 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **MSS**
 
-1. User chooses to find a flashcard. 
+1. User chooses to find a flashcard.
 2. User enters the keyword they would like to search for.
-3. Bagel finds the flashcard and shows a list of flashcards that contain that keyword. 
-
-Use case ends.
+3. Bagel finds the flashcard and shows a list of flashcards that contain that keyword.
+  Use case ends.
 
 
 **Extensions**
 
-* 2a. Bagel detects an error in the entered data.
-    * 2a1. Bagel requests for the correct data.
-    * 2a2. User enters new details.<br>
-      Steps 2a1-2a2 are repeated until the data entered are correct.
-
-Use case resumes from step 3.
+2a. Bagel detects an error in the entered data.
+    2a1. Bagel requests for the correct data.
+    2a2. User enters new details.
+    Steps 2a1-2a2 are repeated until the data entered are correct.
+    Use case resumes from step 3.
```

Comment:
Similarly, it'll end up as one long line here if you don't use the `*` and `<br>`.